forgit.plugin.zsh

```diff
@@ -191,7 +191,7 @@ forgit::rebase() {
         $FORGIT_REBASE_FZF_OPTS
     "
     commit=$(eval "$cmd" | FZF_DEFAULT_OPTS="$opts" fzf | eval "$forgit_extract_sha")
-    [[ -n "$commit" ]] && git rebase -i "$commit"
+    [[ -n "$commit" ]] && git rebase -i "$commit^"
```

Comment:
Does this work when it is the first commit? Or we can work around like this:

https://github.com/wfxr/forgit/blob/bd7942f6f7d99ee1e2d45d24ed40fb6f40f19a57/forgit.plugin.zsh#L215-L220

Reply:
Oh yes, brilliant idea! I was already thinking about how we could add support for rebasing on the very first commit. :-) I'm gonna add this!